packages/shared/src/components/streak/popup/ReadingStreakPopup.tsx

```diff
@@ -82,10 +99,13 @@ interface ReadingStreakPopupProps {
   fullWidth?: boolean;
 }
 
+const timezoneSettingsHref = '/account/notifications?s=timezone';
```

Comment:
just in case we want to do something special when user comes from timezone prompt in the future